src/util/postgres_client.js

```diff
@@ -1496,22 +1497,27 @@ class PostgresClient extends EventEmitter {
             }
         };
 
-        const postgres_port = parseInt(process.env.POSTGRES_PORT || '5432', 10);
 
-        if (process.env.POSTGRES_CONNECTION_STRING) {
+        if (process.env.POSTGRES_CONNECTION_STRING_PATH) {
             /** @type {import('pg').PoolConfig} */
             this.new_pool_params = {
-                connectionString: process.env.POSTGRES_CONNECTION_STRING,
+                connectionString: fs.readFileSync(process.env.POSTGRES_CONNECTION_STRING_PATH, "utf8"),
                 ...params,
             };
         } else {
+            // get the connection configuration. first from env, then from file, then default
+            const host = process.env.POSTGRES_HOST || fs_utils.try_read_file_sync(process.env.POSTGRES_HOST_PATH) || '127.0.0.1';
```

Comment:
Shouldn't they be all in the same file? we will have different file for every env?

Reply:
This is the way secrets are mounted in kubernetes. Each key in the secret is mounted in a separate file

Reply:
We can save a a JSON string under the same key if we want, but I guess that as it's only few of them, it's not that bad.
Maybe don't use 4 different envs, and instead use one for all, and under them we can use user/password/dbname and port

Reply:
after a discussion with @dannyzaken decided to approve.